New package: Borsh v0.1.0

JuliaRegistrator opened the pull request:
- Registering package: Borsh
- Repository: https://github.com/ozgb/Borsh.jl
- Created by: @ozgb
- Version: v0.1.0
- Commit: 9ca539c046cff1e9d6394e9eafa2b4eda7e6fa0b
- Reviewed by: @ozgb
- Reference: https://github.com/ozgb/Borsh.jl/commit/9ca539c046cff1e9d6394e9eafa2b4eda7e6fa0b#commitcomment-152310683
- Description: Borsh serialization/deserialization for Julia
<!-- bf0c69308befbd3ccf2cc956ac8a46712550b79fc9bfb5e4edf8f833f05f4c18b06eddad8845b45beb9f45c2b8020dd6d3ebc923c3bcee33a1d430c92b71d47c69a4c25a81e0982010129fa1454273002556d1106beeaa2ac9daae709013f3b8403aa3071f87729c60200cbf2716aff4a6c26b4c10340c3f90579fb30f199647fd4077d29b6533d2b951e9eb20168de84e473e5a989fc7573284cda9316a74fc5ba60357cdcccbf9aaa4d0c5f7732797b1ecb336b19568cd7055fe62517bd7cedd3c7bd5746b3a8923153f8ed04ac134 -->

ozgb commented:
Could I get an manual merge of my package please? I'm getting an error on the name - although it is similar to the conflicts listed, the name matches the name of the serialization spec I've implemented

JuliaRegistrator committed:
New package: Borsh v0.1.0

UUID: e0ca245e-e2d4-4924-a0cc-c0f16c675cd2
Repo: https://github.com/ozgb/Borsh.jl.git
Tree: 70521771de341e5ae37b06c3bcb54e979f9fbccf

Registrator tree SHA: 17aec322677d9b81cdd6b9b9236b09a3f1374c6a
